kickstart-london.tex

```diff
@@ -0,0 +1,167 @@
+% Footnotes used inline in the text.
+\newcommand{\manualhandlingfootnote}{\footnote{\url{http://www.hse.gov.uk/pubns/indg143.pdf}}}
+\newcommand{\chargingfootnote}{\footnote{\url{https://studentrobotics.org/docs/kit/batteries/}}}
+\newcommand{\estatesfacilitiesfootnote}{\footnote{Estates and Facilities: \url{http://www.southampton.ac.uk/estates/}}}
+\newcommand{\hscfootnote}{\footnote{Health and Safety Coordinator; for the duration of this event, this role will be filled by Andrew Barrett-Sprot (\email{abarrett-sprot@studentrobotics.org}).}}
+
+
+\newcommand{\groupname}{Student Robotics}
+\newcommand{\assessorname}{Andrew Barrett-Sprot}
+\newcommand{\assessoremail}{abarrett-sprot@studentrobotics.org}
+\newcommand{\assessmentdate}{October 20, 2018}
+
+
+\newcommand{\activityname}{Student Robotics 2019 - Kickstart}
+\newcommand{\activitydate}{November 10, 2018}
+\newcommand{\activitytime}{All day}
+\newcommand{\activitylocation}{
+    Office for Thread, 1 Alie St, London E1 8DE, UK
```

Comment:
Perhaps "Thread's Offices"? "Office for Thread" sounds odd to me. If this is meant to be an address, then maybe drop the "Offices" part entirely?